docker/nonvoting_mixnet/conf/provider1/katzenpost.toml

```diff
@@ -59,28 +59,26 @@
 #
 
 [Provider]
-EnableEphemeralClients = true
-TrustOnFirstUse = true
-
-   [[Provider.CBORPluginKaetzchen]]
+  EnableServiceProvider = true
+  [[Provider.CBORPluginKaetzchen]]
     Capability = "echo"
-    Endpoint = "echo"
+    Endpoint = "+echo"
```

Comment:
what was happening prior to adding +?

Reply:
it doesn't matter what we call it. it's just a string in a database

Reply:
no. what I meant was that this change on this line isn't needed for things to work. It works if it's set to "echo" and also works if set to "+echo". I hope that this is clear now. The endpoint was being found and there was no reason to think it wasn't. Nothing wrong with this line if we do not change it. Jesus.